CONTRIBUTORS.txt

```diff
@@ -0,0 +1,26 @@
+For the purpose of tracking copyright, this is the list of individuals and
+organizations who have contributed source code to the Swift Logging API.
+
+For employees of an organization/company where the copyright of work done
+by employees of that company is held by the company itself, only the company
+needs to be listed here.
+
+## COPYRIGHT HOLDERS
+
+- Apple Inc. (all contributors with '@apple.com')
+
+### Contributors
+
+- David Hart <david@hartbit.com>
+- Iain Smith <iainsmith@users.noreply.github.com>
+- Ian Partridge <i.partridge@uk.ibm.com>
+- Johannes Weiss <johannesweiss@apple.com>
+- Konrad `ktoso` Malawski <konrad.malawski@project13.pl>
```

Comment:
@ktoso  please confirm in a comment that you are happy with your code being added to a Swift Logging API package under the terms of the Apache 2 license (see `LICENSE.txt`)

Reply:
Confirmed 👍

Reply:
Please see this PR too https://github.com/weissi/swift-server-logging-api-proposal/pull/53 with added mail mappings for myself

Reply:
Merged